app/uk.gov.hmrc.helptosaveproxy/config/AppConfig.scala

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
`scalastyle-config.xml`  is missing , its needed i guess, if so, please add this too

Reply:
Added! Thanks for spotting that.